discord/__init__.py

```diff
@@ -69,6 +69,7 @@ class VersionInfo(NamedTuple):
     serial: int
 
 
-version_info: VersionInfo = VersionInfo(major=2, minor=0, micro=0, releaselevel='alpha', serial=0)
+version_info: VersionInfo = VersionInfo(
```

Comment:
It  was better in one line

Reply:
Doesn't work. Black splits it again and again. I have done another thing now, will show u when I will commit